frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -94,6 +98,7 @@ interface IManageHostsProps {
   route: RouteProps;
   router: InjectedRouter;
   params: Params;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   location: any; // no type in react-router v3
```

Comment:
Can you add these comments to `location: any` on the ManagePoliciesPage? That's the only other place in the codebase that doesn't have the linter comment